lib/query-builders/ntco.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
       .where(builder => {
         builder.whereJsonSupersetOf('data', { model: 'pil', action: 'grant' });
         builder.orWhereJsonSupersetOf('data', { model: 'pil', action: 'transfer' });
+        builder.orWhereJsonSupersetOf('data', { model: 'pil', action: 'review' });
```

Comment:
Is there any kind of `pil` task that might have one of the specified statuses that we _wouldn't_ want in this query?

If not then probably we should consolidate these into a single `{ model: 'pil' }` query for efficiency.

Reply:
revoke

Reply:
I'd still probably get rid of the action criteria and reduce this to a single clause.

It'll make the query faster, and task-list queries need all the help they can get, and on the outside chance that a revocation ended up in a `awaiting-endorsement` state then it's possibly still the right thing to include it.